<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.442, y1:0.568, x2:1, y2:0.767, stop:0.589474 rgba(25, 25, 25, 255), stop:1 rgba(126, 10, 10, 255));</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="main_menu">
       <widget class="QPushButton" name="SpamDetectionBTN">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>330</y>
          <width>221</width>
          <height>140</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qradialgradient(spread:reflect, cx:0.473684, cy:0.347, radius:1.118, fx:0.478526, fy:0.342, stop:0.331579 rgba(246, 0, 0, 255), stop:0.626316 rgba(150, 0, 0, 255));
font: 20pt &quot;Trebuchet MS&quot;;
text-align: center;
border: 6px solid;
border-radius: 45px;
color: rgb(255, 255, 255);
border-color: rgb(135, 0, 0);
}

QPushButton:hover{
background-color: rgb(0, 0, 0);
color: rgb(246, 0, 0);
border: 4px solid;
border-color: rgb(246, 0, 0);
}

QPushButton:pressed{
background-color: rgb(44, 44, 44);
color: rgb(189, 0, 0);
border: 4px solid;
border-color: rgb(189, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Spam
 Detection</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="logo">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>751</width>
          <height>161</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#logo{
border-image: url(:/logo/assets/logo.png);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="SL_submission">
       <widget class="QPushButton" name="GoBackBTN">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qradialgradient(spread:reflect, cx:0.473684, cy:0.347, radius:1.118, fx:0.478526, fy:0.342, stop:0.331579 rgba(246, 0, 0, 255), stop:0.626316 rgba(150, 0, 0, 255));
font: 15pt &quot;Trebuchet MS&quot;;
text-align: center;
border: 3px solid;
border-radius: 30px;
color: rgb(255, 255, 255);
border-color: rgb(135, 0, 0);
}

QPushButton:hover{
background-color: rgb(0, 0, 0);
color: rgb(246, 0, 0);
border-color: rgb(246, 0, 0);
}

QPushButton:pressed{
background-color: rgb(44, 44, 44);
color: rgb(189, 0, 0);
border-color: rgb(189, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="UserInput">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>250</y>
          <width>761</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit{
background-color: rgb(239, 239, 239);
border: 4px solid;
font: 18pt &quot;Rockwell&quot;;
border-color: black;
border-radius: 13px;
}

QTextEdit:focus{
border-color: rgb(190, 2, 2);
background-color: rgb(255, 220, 220);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="documentTitle">
         <string/>
        </property>
        <property name="markdown">
         <string/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Rockwell'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="InfoLabel">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>551</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#InfoLabel{
font: 15pt &quot;Rockwell&quot;;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
}</string>
        </property>
        <property name="text">
         <string>Please paste below the message to be checked</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="SubmitBTN">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>510</y>
          <width>181</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qradialgradient(spread:reflect, cx:0.473684, cy:0.347, radius:1.118, fx:0.478526, fy:0.342, stop:0.331579 rgba(246, 0, 0, 255), stop:0.626316 rgba(150, 0, 0, 255));
font: 20pt &quot;Trebuchet MS&quot;;
text-align: center;
border: 6px solid;
border-radius: 45px;
color: rgb(255, 255, 255);
border-color: rgb(135, 0, 0);
}

QPushButton:hover{
background-color: rgb(0, 0, 0);
color: rgb(246, 0, 0);
border: 4px solid;
border-color: rgb(246, 0, 0);
}

QPushButton:pressed{
background-color: rgb(44, 44, 44);
color: rgb(189, 0, 0);
border: 4px solid;
border-color: rgb(189, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
       <widget class="QLabel" name="WarningLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>620</y>
          <width>451</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#WarningLabel{
color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="SL_results">
       <widget class="QLabel" name="MessageLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#MessageLabel{
font: 20pt &quot;Rockwell&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
}</string>
        </property>
        <property name="text">
         <string>Your Message:</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="CheckedMessage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>411</width>
          <height>511</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit{
border: 4px solid;
font: 18pt &quot;Rockwell&quot;;
border-color: rgb(144, 31, 31);
background-color: rgb(239, 239, 239);
border-radius: 13px;
}
</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="ResultsLabel">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>110</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#ResultsLabel{
font: 20pt &quot;Rockwell&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
}</string>
        </property>
        <property name="text">
         <string>Check results:</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Results">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>160</y>
          <width>411</width>
          <height>511</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit{
border: 4px solid;
font: 18pt &quot;Rockwell&quot;;
border-color: rgb(144, 31, 31);
background-color: rgb(239, 239, 239);
border-radius: 13px;
}
</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="GoBackBTN_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qradialgradient(spread:reflect, cx:0.473684, cy:0.347, radius:1.118, fx:0.478526, fy:0.342, stop:0.331579 rgba(246, 0, 0, 255), stop:0.626316 rgba(150, 0, 0, 255));
font: 15pt &quot;Trebuchet MS&quot;;
text-align: center;
border: 3px solid;
border-radius: 30px;
color: rgb(255, 255, 255);
border-color: rgb(135, 0, 0);
}

QPushButton:hover{
background-color: rgb(0, 0, 0);
color: rgb(246, 0, 0);
border-color: rgb(246, 0, 0);
}

QPushButton:pressed{
background-color: rgb(44, 44, 44);
color: rgb(189, 0, 0);
border-color: rgb(189, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/logo_res.qrc"/>
 </resources>
 <connections/>
</ui>
